benchmarks/README.md

```diff
@@ -26,10 +26,10 @@ The benchmarking script requires the following to run:
 
 ## Models
 
-The pre-loaded models for the benchmark can be mostly found in the [MMS model zoo](https://github.com/awslabs/mxnet-model-server/blob/master/docs/model_zoo.md).  We currently support the following:
-- [resnet: ResNet-18 (Default)](https://github.com/awslabs/mxnet-model-server/blob/master/docs/model_zoo.md#resnet-18)
-- [squeezenet: SqueezeNet V1.1](https://github.com/awslabs/mxnet-model-server/blob/master/docs/model_zoo.md#squeezenet_v1.1)
-- [lstm: lstm-ptb](https://github.com/awslabs/mxnet-model-server/blob/master/docs/model_zoo.md#lstm-ptb)
+The pre-loaded models for the benchmark can be mostly found in the [MMS model zoo](https://github.com/awslabs/multi-model-server/blob/master/docs/model_zoo.md).  We currently support the following:
```

Comment:
These links wont work until the project name is changed.

Reply:
I purposefully didn't just replace everything as not everything has actually been updated (docker containers, s3 buckets, etc.). Above is fine, will change project name once merged, unless you think we should do it before.